content/en/blog/2022/2022-02-04-sig-multicluster-spotlight-blog.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: blog
+title: "Spotlight on SIG Multicluster"
+date: 2022-02-04
+slug: sig-multicluster-spotlight-2022
+---
+
+**Authors:** Dewan Ahmed (Aiven) and Chris Short (AWS)
+
+## Introduction
+
+[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) is the SIG focused on how Kubernetes concepts are expanded and used beyond the cluster boundary. Historically, Kubernetes resources only interacted within that boundary - KRU or Kubernetes Resource Universe (not an actual Kubernetes concept). Kubernetes clusters, even now, don't really know anything about themselves or, about other clusters. Absence of cluster identifiers is a case in point. With the growing adoption of multicloud and multicluster deployments, the work SIG Multicluster doing is gaining a lot of attention. In this blog, [Jeremy Olmsted-Thompson, Google](https://twitter.com/jeremyot) and [Chris Short, AWS](https://twitter.com/ChrisShort) discuss the interesting problems SIG Multicluster is solving and how you can get involved. Their initials **JOT** and **CS** will be used for brevity.
```

Comment:
```suggestion
[SIG Multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) is the Special Interest Group (SIG) focused on how Kubernetes concepts are expanded and used beyond the cluster boundary. Historically, Kubernetes resources only interacted within that boundary - KRU or Kubernetes Resource Universe (not an actual Kubernetes concept). Kubernetes clusters, even now, don't really know anything about themselves or, about other clusters. Absence of cluster identifiers is a case in point. With the growing adoption of multicloud and multicluster deployments, the work SIG Multicluster doing is gaining a lot of attention.

In this article, [Jeremy Olmsted-Thompson, Google](https://twitter.com/jeremyot) and [Chris Short, AWS](https://twitter.com/ChrisShort) discuss the interesting problems SIG Multicluster is solving and how you can get involved. Their initials **JOT** and **CS** will be used for brevity.
```

&hellip;explaining what SIG means, and also calling it an article. This is an article that appears in a blog - if you prefer using the word “blog” to mean both the set of articles and each individual article, you can omit that fix.